sudo

```diff
@@ -0,0 +1 @@
+151.101.4.162 registry.npmjs.org /etc/hosts
```

Comment:
I don't think you need to add this. :)

Reply:
I couldn't find this in the diff. Can you tell me the file name. If its related to something else can you explain please!